common/storage/sql/sqlite3/storage.c

```diff
@@ -33,39 +33,53 @@ retcode_t iota_stor_destroy(const connection_t* const conn) {
   return destroy_connection(conn);
 }
 
+/*
+ * Compression functions
+ */
+
+static int column_compress(sqlite3_stmt* statement, size_t index,
```

Comment:
can we rename to `column_compress_bind` to indicate we are binding?

Reply:
Sure